lighthouse-core/audits/accessibility/label.js

```diff
@@ -33,6 +33,9 @@ class Label extends AxeAudit {
       category: 'Accessibility',
       name: 'label',
       description: 'Every form element has a label',
+      helpText: 'Although the purpose of each form element may be obvious ' +
```

Comment:
Suggest rewording to "Labels ensure that form controls are announced properly by assistive technology, such as a screen reader"